game/scripts/npc/abilities/windrunner_focusfire.txt

```diff
@@ -42,7 +42,7 @@
       "02"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "focusfire_damage_reduction"                      "-50 -40 -30 -20 -10"
+        "focusfire_damage_reduction"                      "-50 -40 -30 -15 0"
```

Comment:
better scaling with removal of talent, with talent prior to change it ended up at 6% bonus dmg (wat). now it's 0% max.

Reply:
I admit it's confusing sometimes that WR ult at OAA levels with the talent gives bonus damage, but I don't see why is that a problem and why would that need a nerf? Talent change to armor corruption still doesn't make sense to me.

Reply:
better scaling for removing the talent is kind of weird, if level 3 is unchanged then its just a nerf (the gap between 3 4 and 5 is massive). I think the armor corruption talents are all shit

Reply:
> better scaling for removing the talent is kind of weird, if level 3 is unchanged then its just a nerf (the gap between 3 4 and 5 is massive). I think the armor corruption talents are all shit

each armor provides effectively 6% EHP. 4 armor removes effectively 24% EHP, This is a buff regardless.

Reply:
with this change 
-at level 20 any enemy with more than 4 armor or less than 2 armor takes less damage 
-at level 4 ult any enemy with more than 19 armor or less than -2 armor takes less damage
-at level 5 ult any enemy with more than 54 armor or less than -10 armor takes less damage 

feel free to check my math and call me a stupid noob

Reply:
yeah okay technically this is a buff, idk how you make this look pretty without keeping the old talent and/or nerfing wind.

Reply:
I think my proposed changes are the best to make it look pretty and provide wind some flexibility outside of her ult in terms of dealing damage. Also helps with boss taking as a team and expands her role a bit. I know she's a strong hero at times but she does need a slight improvement I feel.

Reply:
It is a lvl 5 buff but a lvl 20 nerf.

Reply:
> It is a lvl 5 buff but a lvl 20 nerf.

I'm ok with this knowing where wind is at rn.